### PR TITLE
Check for AFK only inGame status sessions

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR 
Fixes an issue when a player is in a queue, for example, and his status is Connected, and he doesn't do any input actions 👉 👈 

**Screenshots**

**Changelog**

### CODE REVIEW

File: Content.Server/Afk/AFKSystem.cs [hunk 1]
```diff
@@ -85,6 +85,8 @@ public override void Update(float frameTime)
 
         foreach (var session in Filter.GetAllPlayers())
         {
+            if (session.Status != SessionStatus.InGame) return;
```
Comment (metalgearsloth) [inline Content.Server/Afk/AFKSystem.cs:88]: should be a continue.

Also when are people marked as ingame.

**Comment by Megmeehey** [comment]
AFKSystem should also have checks, if player is in crit, since ppl are unable to do anything in that status

**Comment by Morb0** [comment]
> AFKSystem should also have checks, if player is in crit, since ppl are unable to do anything in that status

It is possible to find a million cases that way, is it better that the client would send a message about entering the AFK?
The client gets the possibility to abuse it, but like seriously...?

File: Content.Server/Afk/AFKSystem.cs (hunk 1 above)
Comment (Morb0) [reply Content.Server/Afk/AFKSystem.cs]: Fixed continue

**Review by AJCM-git — APPROVED** [review]

**Comment by AJCM-git** [comment]
can someone rerun tests on this?

**Pull request merged**